src/frontends/pytorch/src/op/var_mean.cpp

```diff
@@ -76,6 +78,43 @@ OutputVector translate_var_mean(const NodeContext& context) {
     return {var, mean};
 };
 
+OutputVector translate_var_mean_fx(const NodeContext& context) {
+    num_inputs_check(context, 1, 4);
+    auto data = context.get_input(0);
+    bool unbiased = false;
+    auto num_elements = numel(context, data);
+    std::shared_ptr<ov::Node> mean, t_mean;
+    ov::Output<ov::Node> axes;
+
+    axes = context.get_input(1);
+    auto axis_0 = context.mark_node(v0::Constant::create(element::i32, Shape{}, {0}));
+    mean = context.mark_node(std::make_shared<v1::ReduceMean>(data, axes, true));
+    t_mean = context.mark_node(std::make_shared<v1::ReduceMean>(data, axes, true));
+    auto reduced_dims = context.mark_node(std::make_shared<v3::ShapeOf>(data, element::i32));
+    auto zero = context.mark_node(v0::Constant::create(element::i32, Shape{}, {0}));
+    reduced_dims = context.mark_node(std::make_shared<v8::Gather>(reduced_dims, axes, zero));
+    num_elements = context.mark_node(std::make_shared<v1::ReduceProd>(reduced_dims, zero, false));
+
+    auto sub_v = context.mark_node(std::make_shared<v1::Subtract>(data, t_mean));
```

Comment:
is it possible to reuse some code between translators? Looks very similar to ts translator, except output replacing and first input check